### PR TITLE
Add remarks on disposable ctor parameter to System.Net.Http.StreamCon…

### LINKED ISSUE #3339: Add information about StreamContent disposing the content stream
The content stream used to initialize `StreamContent` is disposed when the `StreamContent` instance is disposed.

This should be mentioned on the documentation.

https://docs.microsoft.com/en-us/dotnet/api/system.net.http.streamcontent.-ctor
https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Net.Http/StreamContent.xml

### PR DESCRIPTION
**Pull request by JJS**
…tent (#3339)

## Summary

Added remarks for the System.Net.Http.StreamContent constructors disposing the passed stream.

Fixes #3339

### CODE REVIEW

**Comment by BillWagner** [comment]
ping @carlossanlop

**Comment by aik-jahoda** [comment]
I can't see the change in the doc preview https://review.docs.microsoft.com/en-us/dotnet/api/system.net.http.streamcontent.-ctor?branch=pr-en-us-3357&view=netcore-3.1#System_Net_Http_StreamContent__ctor_System_IO_Stream_

**Comment by carlossanlop** [comment]
@aik-jahoda It seems the build output was too old and got deleted. If you expand the "Show all checks" section, then click on "Details" in the OpenPublishing.Build row, the page is gone.

The build passed though, so the formatting was correct. And the xref links look correct to me too.

If you want to double check the output, we can run the build again. @dotnet/docs do you know how to re-run the build?

**Comment by gewarren** [comment]
You just close/reopen the PR to trigger a new build. I'll do that now.

File: xml/System.Net.Http/StreamContent.xml [hunk 1]
```diff
@@ -66,7 +66,14 @@
       <Docs>
         <param name="content">The content used to initialize the <see cref="T:System.Net.Http.StreamContent" />.</param>
         <summary>Creates a new instance of the <see cref="T:System.Net.Http.StreamContent" /> class.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The <xref:System.Net.Http.StreamContent> object calls <xref:System.IO.Stream.Dispose> on the provided <xref:System.IO.Stream> object when <xref:System.Net.Http.StreamContent.Dispose%2A?displayProperty=nameWithType> is called.  
+
```
Comment (adegeo) [inline xml/System.Net.Http/StreamContent.xml:74]: The formatting on these should be fixed.... Whatever ports these comments in adds double spaces to the ends of the line, indents the starting paragraph for no reason, and doesn't properly space the headers. All of these should be fixed before merging.

@carlossanlop FYI

```suggestion
  
## Remarks

The <xref:System.Net.Http.StreamContent> object calls <xref:System.IO.Stream.Dispose> on the provided <xref:System.IO.Stream> object when <xref:System.Net.Http.StreamContent.Dispose%2A?displayProperty=nameWithType> is called.

```

File: xml/System.Net.Http/StreamContent.xml [hunk 2]
```diff
@@ -98,7 +105,14 @@
         <param name="content">The content used to initialize the <see cref="T:System.Net.Http.StreamContent" />.</param>
         <param name="bufferSize">The size, in bytes, of the buffer for the <see cref="T:System.Net.Http.StreamContent" />.</param>
         <summary>Creates a new instance of the <see cref="T:System.Net.Http.StreamContent" /> class.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The <xref:System.Net.Http.StreamContent> object calls <xref:System.IO.Stream.Dispose> on the provided <xref:System.IO.Stream> object when <xref:System.Net.Http.StreamContent.Dispose%2A?displayProperty=nameWithType> is called.
```
Comment (gewarren) [inline xml/System.Net.Http/StreamContent.xml:112]:
```suggestion
The <xref:System.Net.Http.StreamContent> object calls <xref:System.IO.Stream.Dispose> on the provided <xref:System.IO.Stream> object when <xref:System.Net.Http.StreamContent.Dispose%2A?displayProperty=nameWithType> is called.
```

**Pull request merged**